src/index.cpp

```diff
@@ -1023,8 +1078,12 @@ namespace diskann {
       boost::dynamic_bitset<> & inserted_into_pool_bs) {
     const T *node_coords = _data + _aligned_dim * node_id;
 
-    if (init_ids.size() == 0)
+    if (init_ids.size() == 0) {
       init_ids.emplace_back(_start);
```

Comment:
when you rebase to main, this code that set init_ids could move to search_for_point_and_add_link function. We removed get_expanded_node function